torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -146,6 +152,13 @@ std::unique_ptr<mobile::Function> FlatbufferLoader::parseFunction(
 
   std::unordered_set<std::string> unsupported_op_names;
   const int64_t model_version = 0x6L;
+
+  appendUpgraderFunctions(function.get());
+  // 2. Decides if upgrader is needed
+  const uint32_t operator_version = module_->version();
+  bool use_upgrader =
```

Comment:
I believe this is not necessary because the model version cannot go beyond kProducedFileFormatVersion

Reply:
I copied from here: https://github.com/pytorch/pytorch/blob/c53b3ed20fc84438b5354bc81d410b4d1a1c4e4f/torch/csrc/jit/mobile/import.cpp#L389

I guess it's possible because the binary writing the file and the binary reading the file are not the same version.